backend/packages/Upgrade/src/api/services/ExperimentAssignmentService.ts

```diff
@@ -254,14 +254,7 @@ export class ExperimentAssignmentService {
     );
 
     if (filteredExperiments.length) {
-      let monitoredDocument: MonitoredDecisionPoint = await this.monitoredDecisionPointRepository.findOne({
-        where: {
-          site: site,
-          target: target,
-          user: userId,
-        },
-        relations: ['user'],
-      });
+      // individual/group level exclusions:
 
       if (experimentId && experiments.length > 0) {
```

Comment:
We should not use `experimentId` and `experiments` here. We should use `filteredExperiments` to get the eexperiment.